### ISSUE TITLE
Edit tags of multiple files

### ISSUE THREAD

**Issue by mklein994** [vanilla-music/vanilla-music-tag-editor#9]
I haven't found out if this is already implemented, but it would speed up batch processing if I could select multiple songs and assign an album or artist to them.

**Comment by Adonai** [comment]
Depends on vanilla-music/vanilla actually, I haven't seen a way to select multiple songs there.
Long click on a song gives you context menu, not selection frame.

**Comment by tbdal** [comment]
This would be great. Will you foresee that in future releases?

**Comment by Adonai** [comment]
No. Vanilla player still doesn't have the ability to select multiple songs and I have other pressing tasks in my life.
